docs/src/development_environment.md

```diff
@@ -62,7 +62,7 @@ This is based on [`nix-fast-build`](https://github.com/Mic92/nix-fast-build#read
 ## Documentation
 
 The documentation in the `docs` subtree gets
-[published](https://stylix.danth.me) automatically, using GitHub Actions.
+[published](https://nix-community.github.io/stylix) automatically, using GitHub Actions.
```

Comment:
```suggestion
[published](https://nix-community.github.io/stylix) automatically, using GitHub
Actions.
```
In order not to exceed the 80-column limit. Otherwise, I reckon that the next person that edits this file will have to do it then, right?

Reply:
Ah, I was wrong. This file is exempt.